.github/workflows/push_to_gcloud.yml

```diff
@@ -43,6 +43,17 @@ jobs:
         python script/agencies_validate.py \
           './airflow/data/agencies.filled.yml' \
           './airflow/data/schema_agencies_entry.yml'
+    - name: Fill in headers.yml API keys
+      run: |
+        python script/yml_convert.py \
+          './airflow/data/headers.yml' \
+          './airflow/data/headers.filled.yml' \
+          './airflow/data/secrets.csv'
+    - name: Validate headers.yml
+      run: |
+        python script/headers_validate.py \
+          './airflow/data/headers.filled.yml' \
+          './airflow/data/schema_headers_entry.yml'
 
     - if: ${{ github.repository == 'cal-itp/data-infra' }}
       uses: google-github-actions/setup-gcloud@master
```

Comment:
I can't comment on the particular line directly, but in the later step `Push agencies data to google cloud and prod services`, there is some logic that seems to set an environment variable in a kubernetes configuration. I believe we need to do the same thing for `headers.filled.yml` as was done for `agencies.filled.yml`.